common-crafting.lic

```diff
@@ -197,6 +197,51 @@ module DRCC
     true
   end
 
+  def get_adjust_tongs(usage, bag, bag_items, belt, adjustable_tongs)
```

Comment:
Can you rename this to `get_adjust_tongs?` since it is a predicate?

Reply:
I don't know. I can try!